src/tests/ftest/scripts/main.sh

```diff
@@ -179,9 +179,15 @@ if [ "$(lsb_release -s -i)" = "CentOS" ]; then
 else
     process_cores=""
 fi
+
+# Clean stale job results
+if [ -d "$logs_prefix/ftest/avocado/job-results" ]; then
+    rm -rf "$logs_prefix/ftest/avocado/job-results"
```

Comment:
Does shell check not complain about this?
```suggestion
    rm -rf "${logs_prefix}/ftest/avocado/job-results"
```

Reply:
Braces around variable names are not a requirement by shells or even shellcheck.  They are only required when a variable name is abutted up against a string that is valid variable name characters.  I.e. `echo "${my_var}iable"`.